src/Raven.Server/Documents/Queries/MoreLikeThis/MoreLikeThisQueryServerSide.cs

```diff
@@ -58,6 +58,10 @@ public static MoreLikeThisQueryServerSide Create(HttpContext httpContext, int pa
 
                         includes.Add(item.Value[0]);
                     }
+                    else if (string.Equals(item.Key, "document", StringComparison.OrdinalIgnoreCase))
```

Comment:
Why aren't we passing this as a POST request?

Reply:
So we shouldn't support GET /queries for MoreLikeThis queries?
(https://github.com/ravendb/ravendb/blob/v4.0/src/Raven.Server/Documents/Handlers/QueriesHandler.cs#L83)